public/components/cards/27.html

```diff
@@ -0,0 +1,48 @@
+<a class="block bg-[#fefefe] shadow-xl rounded-xl text-center p-5" href="">
+  <h5
+    class="text-3xl font-bold mb-1 text-transparent bg-clip-text bg-gradient-to-r from-blue-600 via-blue-500 to-blue-600"
+  >
+    Basic Plan
+  </h5>
+
+  <h6 class="text-slate-900 text-sm">
+    Perfect for private or small business websites!
+  </h6>
+
+  <div class="mt-3 text-slate-900">
+    <span class="text-2xl">$</span>
+    <h6 class="inline text-5xl font-bold">9</h6>
+    <span class="text-xs"> / month</span>
+    <p class="text-xs text-slate-700">Billed annually</p>
+  </div>
+
+  <ul class="mt-10 text-slate-900 leading-6">
+    <li class="mb-3">
+      <span aria-hidden="true" class="text-blue-600 mr-2">&check;</span>Single
+      website
+    </li>
+    <li class="mb-3">
+      <span aria-hidden="true" class="text-blue-600 mr-2">&check;</span>One
+      click WordPress installs
+    </li>
+    <li class="mb-3">
+      <span aria-hidden="true" class="text-blue-600 mr-2">&check;</span
+      >Unmetered bandwith
+    </li>
+    <li class="mb-3">
+      <span aria-hidden="true" class="text-blue-600 mr-2">&check;</span> Free
+      SSL certificate
+    </li>
+    <li class="mb-3">
+      <span aria-hidden="true" class="text-blue-600 mr-2">&check;</span>30 day
+      money back guarantee
+    </li>
```

Comment:
Need to add a whitespace between each `<li></li>` element and move the text onto it's own line.

Use a check from [heroicons.com](heroicons.com).

Replace `mb-3` on each `<li>` with adding `space-y-3` to the parent element. This will fix the spacing issue between the last item and the button.

Reply:
And let's use dummy text for now.

Reply:
Just need to add some whitespace between the `<li></li>` elements like this.

```html
<li></li>

<li></li>

<li></li>
```

Makes it a little easier to read.

The dummy text looks good, I'll have a think of what we can replace it with that's less WordPress-y.